test/test_meta.py

```diff
@@ -662,7 +662,6 @@ def run_meta_crossref(
     torch.functional.einsum : {bf16, c128, f64, f32, f16, c64},
     torch.functional.tensordot : {bf16, i8, i64, u8, c128, f64, i16, f32, i32, c64},
     torch.inner : {bf16, i8, i64, u8, c128, f64, i16, f32, i32, c64},
-    torch.linalg.lu_solve : {c128, c64},
```

Comment:
Unlocks test_meta and test_proxy_tensor tests, but test_aotdispatch requires adding support for lu_unpack first.